api/internal/handler/route/route.go

```diff
@@ -219,6 +220,10 @@ func (h *Handler) List(c droplet.Context) (interface{}, error) {
 				return false
 			}
 
+			if input.Status != "" && fmt.Sprintf("%d", obj.(*entity.Route).Status) != input.Status {
```

Comment:
I recommend using `strconv.Itoa` instead of `fmt.Sprintf`. Because the former is faster than the latter in the benchmark test.

Reply:
Thank you for your suggestion. I'll revise it right away